### ISSUE TITLE
Post template broken

### ISSUE THREAD

**Issue by eurich** [SimpleMachines/SMF#77]
tested with a completely new  installation, start a new topic and the following warning appears:

"The following error or errors occurred while posting this message:
Warning: this topic has not been posted in for at least 120 days.
Unless you're sure you want to reply, please consider starting a new topic."

a second bug: the preview and post buttons are shown outside the rounframe container.

Tested with Chrome.

**Comment by live627** [comment]
Why are the buttons floated right, anyway?

**Comment by ghost** [comment]
Second one is sorted on local. Will up it soon.

All the button_submit class are right floated for consistency. They were a bit all over the place in 2.0. Their location in Post.template.php is exactly the same as it was in 2.0.

**Comment by live627** [comment]
I've already fixed the first one, so this can be closed once the display issue is merged into the main repo.
